### ISSUE TITLE
Support for wifimanager and ota? 

### ISSUE THREAD

**Issue by jascdk** [SuperHouse/AirQualitySensorD1Mini#1]
Hi and thanks for sharing your project :) is it possible that you would update the code with WiFi manager and or if so maybe also Arduino OTA . Would be cool to set all the parameters through the WiFi manager web interface and also be able to upload eventual updates over the air :) I’m currently waiting for my parts from AliExpress to complete the project :)

**Comment by jonoxer** [comment]
Definitely. That would be awesome. I haven't added WiFi Manager to a project before so it'll be a good chance to learn how it works. I've done OTA on a few projects already.
